synfig-studio/src/gui/dialogs/about.cpp

```diff
@@ -88,19 +88,27 @@ About::About()
 
 	set_copyright(_("Copyright (c) 2001-2020\nSynfig developers & contributors"));
 	Glib::ustring license =
+		"== Synfig Studio ==\n"
 		"This program is free software; you can redistribute it and/or modify "
 		"it under the terms of the GNU General Public License as published by "
 		"the Free Software Foundation; either version 2 of the License, or "
 		"(at your option) any later version.\n\n"
-
-		"This program is distributed in the hope that it will be useful, "
-		"but WITHOUT ANY WARRANTY; without even the implied warranty of "
-		"MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the "
-		"GNU General Public License for more details.\n\n"
-
+		"This program is distributed in the hope that it will be useful, but "
+		"WITHOUT ANY WARRANTY; without even the implied warranty of "
+		"MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the "
+		"\"GNU General Public License\" for more details.\n\n"
 		"You should have received a copy of the GNU General Public License along "
 		"with this program; if not, write to the Free Software Foundation, Inc., "
-		"51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA or visit: http://www.gnu.org/";
+		"51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA or visit: http://www.gnu.org/\n\n\n"
+		"== Vectorizer ==\n"
+		"Vectorizer feature is ported from OpenToonz open-source animation "
+		"software, which is developed from Toonz, a software originall created "
+		"by Digital Video, S.p.A., Rome Italy.\n"
+		"https://github.com/opentoonz/opentoonz/\n\n"
+		"Vectorizer feature code is licensed under BSD 3-Clause \"New\" or "
+		"\"Revised\" License. Redistribution and use in source and binary forms, "
+		"with or without modification, are permitted provided that conditions "
+		"are met. See the \"The 3-Clause BSD License\" for more details.\n";
```

Comment:
You provided a guide to the license with that statement. Shouldn't we provide a link for "The 3-Clause BSD License" to its website also?
https://opensource.org/licenses/BSD-3-Clause

@morevnaproject please look into this.

@ahmedkrmn I'm all OK with your PR :)

Reply:
@Keyikedalube Thanks, please add the "hacktoberfest-accepted" label to the PR if you think we're done.

Reply:
> Thanks, please add the "hacktoberfest-accepted" label to the PR if you think we're done.

@ice0 Can you please add the label or approve this PR :) Participating PR needs approval if it's not going to be merged early. BTW, only two days left.

Reply:
Done. Please check.

Reply:
Thanks, @ice0!!! Sorry I had to ping you specifically because you're the only maintainer I know mostly active reviewing PRs and issues.